pvacseq/lib/main.py

```diff
@@ -193,6 +210,8 @@ def main(args_input = sys.argv[1:]):
                         type=int,
                         help="length of the peptide sequences in the input FASTA file; default 21",
                         default=21)
+    parser.add_argument('-g', '--cufflinks_genes_tracking_file', help='genes.fpkm_tracking file from Cufflinks')
+    parser.add_argument('-i', '--cufflinks_isoforms_tracking_file', help='isoforms.fpkm_tracking file from Cufflinks')
```

Comment:
The existing parameter names match more closely to the cufflinks output file names:
http://cole-trapnell-lab.github.io/cufflinks/cufflinks/

If I had any suggestion in regard to name, it would be to remove the cufflinks names all together and use something like --gene_exp_file and --transcript_exp_file.  The code would only support Cufflinks FPKM tracking files but the interface would not change if we added support for other expression formats/files/values.